Tools/clinic/libclinic/parse_args.py

```diff
@@ -469,7 +469,14 @@ def parse_pos_only(self) -> None:
                 nargs = 'PyTuple_GET_SIZE(args)'
                 argname_fmt = 'PyTuple_GET_ITEM(args, %d)'
 
-        left_args = f"{nargs} - {self.max_pos}"
+        if self.vararg != NO_VARARG:
+            if self.max_pos == 0:
+                self.declarations = "Py_ssize_t nvararg = %s;" % nargs
+            else:
+                self.declarations = "Py_ssize_t nvararg = Py_MAX(%s - %d, 0);" % (nargs, self.max_pos)
```

Comment:
I'd make this an f-string.

Reply:
Ok.  I'm little lost in which style for string formatting should be used.

BTW, probably Py_MAX could be dropped.

Reply:
> Ok. I'm little lost in which style for string formatting should be used.

Yeah, I totally understand. There is no unified style in Argument Clinic :(

> BTW, probably Py_MAX could be dropped.

Hmkey. Perhaps we should produce an `assert`? I presume we cannot check this during generation.